spyder/widgets/helperwidgets.py

```diff
@@ -442,6 +447,89 @@ def filterAcceptsRow(self, row_num, parent):
             return True
 
 
+class PaneEmptyWidget(QFrame):
+    """Widget to show a pane/plugin functionality description."""
+
+    def __init__(self, parent, icon_filename, text, description):
+        super().__init__(parent)
+        # Image
+        image_path = get_image_path(icon_filename)
+        image = QPixmap(image_path)
+        image_height = int(image.height() * 0.8)
+        image_width = int(image.width() * 0.8)
+        image = image.scaled(
+            image_width, image_height,
+            Qt.KeepAspectRatio, Qt.SmoothTransformation
+        )
+        image_label = QLabel(self)
+        image_label.setPixmap(image)
+        image_label.setAlignment(Qt.AlignCenter)
+        image_label_qss = qstylizer.style.StyleSheet()
+        image_label_qss.QLabel.setValues(border="0px")
+        image_label.setStyleSheet(image_label_qss.toString())
+
+        # Main text
+        text_label = QLabel(text, parent=self)
+        text_label.setAlignment(Qt.AlignCenter)
+        text_label.setWordWrap(True)
+        text_label_qss = qstylizer.style.StyleSheet()
+        text_label_qss.QLabel.setValues(
+            fontSize=DialogStyle.ContentFontSize,
+            border="0px"
+        )
+        text_label.setStyleSheet(text_label_qss.toString())
+
+        # Description text
+        description_label = QLabel(description, parent=self)
+        description_label.setAlignment(Qt.AlignCenter)
+        description_label.setWordWrap(True)
+        description_label_qss = qstylizer.style.StyleSheet()
+        description_label_qss.QLabel.setValues(
+            fontSize="10pt",
+            backgroundColor=SpyderPalette.COLOR_OCCURRENCE_3,
+            border="0px",
+            padding="20px"
+        )
+        description_label.setStyleSheet(description_label_qss.toString())
+
+        # Setup layout
+        pane_empty_layout = QVBoxLayout()
+        pane_empty_layout.addWidget(image_label)
+        pane_empty_layout.addWidget(text_label)
+        pane_empty_layout.addWidget(description_label)
+        pane_empty_layout.addStretch(1)
+        pane_empty_layout.setContentsMargins(0, 0, 0, 0)
```

Comment:
```suggestion
        pane_empty_layout.addStretch(2)
        pane_empty_layout.setContentsMargins(10, 0, 10, 8)
```

* Make the bottom stretch longer than the top one, which makes the content look centered.
* Add left, right and bottom margins so that `description_label` doesn't touch the QFrame borders.